biz.aQute.repository/src/aQute/bnd/repository/maven/provider/IndexFile.java

```diff
@@ -324,13 +324,11 @@ private boolean isMulti(String name) {
 	private Map<Archive, Resource> parseSingle(Archive archive, File single) throws Exception {
 		ResourceBuilder rb = new ResourceBuilder();
 		boolean hasIdentity = rb.addFile(single, single.toURI());
-		if (!hasIdentity) {
-			String name = archive.getWithoutVersion();
-			MavenVersion version = archive.revision.version;
-
-			BridgeRepository.addInformationCapability(rb, name, version.getOSGiVersion(), repo.toString(),
-				Constants.NOT_A_BUNDLE_S);
-		}
+		String name = archive.getWithoutVersion();
+		MavenVersion version = archive.revision.version;
+		BridgeRepository.addInformationCapability(rb, name, version.getOSGiVersion(), repo.toString(),
```

Comment:
Not sure this is wise. Did you check for any places which assume the presence of a bnd.info cap means the artifacts is NOT a bundle? You have changed things from a resource has either a osgi.identity or a bnd.info to where every resource has a bnd.info.